src/commands/database/list.mjs

```diff
@@ -1,19 +1,42 @@
 //@ts-check
 
 import { container } from "../../cli.mjs";
+import { commonQueryOptions } from "../../lib/command-helpers.mjs";
+import { FaunaAccountClient } from "../../lib/fauna-account-client.mjs";
+import { performQuery } from "../eval.mjs";
 
 async function listDatabases(argv) {
-  const profile = argv.profile;
   const logger = container.resolve("logger");
-  const accountClient = container.resolve("accountClient");
-  const accountCreds = container.resolve("accountCreds");
-  const accountKey = accountCreds.get({ key: profile }).accountKey;
-  const databases = await accountClient.listDatabases(accountKey);
+
+  // query the account api
+  const accountClient = new FaunaAccountClient();
+  const databases = await accountClient.listDatabases();
   logger.stdout(databases);
+
+  // query the fauna api
```

Comment:
Why do you need to run `Database.all()` here? Is this just leftover from debugging?

Reply:
yeah just an example usage. i'll get rid of it and update the readme instead